tests/features/ecas-register.feature

```diff
@@ -1,12 +1,22 @@
-@javascript @ecas-login
+@api @ecas-login
 Feature: Register through OE Authentication
   In order to be able to have new users
   As an anonymous user of the system
   I need to be able to go to the registration URL
 
+  @DrupalLogin @BackupAuthConfigs @RebuildRouter
   Scenario: Register
     Given I am an anonymous user
     When I visit "the user registration page"
     # Redirected to the Ecas mockup server.
     And I click "External"
     Then I should see "Create an account"
+
+    Given I am logged in as a user with the "administer authentication configuration" permission
+    When I am on "the Authentication configuration page"
+    And I uncheck "Redirect user registration route to EU Login"
+    Then I press "Save configuration"
+
+    Given I am an anonymous user
+    When I visit "the user registration page"
+    Then I should see "Create new account"
```

Comment:
See this is what I mean. We are changing the value from ON to OFF in the UI, but now we do not have a test that checks if it also works correctly if we change it from OFF to ON. This is a UI test so it makes sense we reopen the form to check that the updated value is displayed correctly and can be toggled back. It can happen that there is a bug with the "default_value" in the form.

This is why I think that using tags to "fix" changes made in the UI are pointless. A complete UI test can restore the values through the UI.